src/data_access/FirebaseAccessObject.java

```diff
@@ -68,4 +72,31 @@ public boolean existsByName(String newProjectName) {
         //TODO: add ways to check if newProjectName exists in db collection
         return true;
     }
+
+    public boolean projectNameExists(String projectName, String meetingName){
+        return true;
+    }
+
+    public void saveMeeting(String projectName, String meetingName){
```

Comment:
I think firebase methods need to be edited later after we agree on how we store the data (@Kosei1227 will look into it by end of this week)